utils/plex.py

```diff
@@ -74,8 +75,41 @@ def remove_user(plex_account, user_email):
         logging.info(f"PLEXAPI: Removed {user_email} from shares on account: {plex_account.email}")
         return True
     except NotFound:
-        logging.error(f"PLEXAPI: Cannot remove {user_email} from shares on account: {plex_account.email}: not sharing with this user")
+        logging.error(
+            f"PLEXAPI: Cannot remove {user_email} from shares on account: {plex_account.email}: not sharing with this user")
         return False
     except Exception as e:
-        logging.error(f"PLEXAPI: Cannot remove {user_email} from shares on account: {plex_account.email} due to unhandled exception: {e}")
+        logging.error(
+            f"PLEXAPI: Cannot remove {user_email} from shares on account: {plex_account.email} due to unhandled exception: {e}")
         return False
+
+
+def get_users(plex_connections, account_email):
+    try:
+        logging.info(f"PLEXAPI: Gettings users from {account_email}")
+        users = {}
+        for account in plex_connections:
+            if account['account'].email == account_email:
+                for user in account['account'].users():
+                    print(user.email, user.username, user.id)
```

Comment:
Are you merging debug print statements into master bro

Reply:
I think I have removed most of the egregious prints